pkg/renovate/task.go

```diff
@@ -117,7 +117,7 @@ func (t *Task) TransformHostRules(ctx context.Context, registrySecret *corev1.Se
 	return hostRules, nil
 }
 
-func (t *Task) GetJobConfig(ctx context.Context, client client.Client, registrySecret *corev1.Secret) (string, error) {
+func (t *Task) GetPipelineRunConfig(ctx context.Context, client client.Client, registrySecret *corev1.Secret) (string, error) {
```

Comment:
This is not used at all at this moment, right? then we don't need to include it.